### PR TITLE
Add SFMC GA release notes

### PR DESCRIPTION
**Pull request by peterpogorski**

### CODE REVIEW

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Microsoft Azure Service Fabric managed clusters general availability release
+
+Azure Service Fabric managed clusters are now generally available. We first introduced Service Fabric managed clusters in October, 2020. Service Fabric managed clusters enable customers to deploy Service Fabric clusters as a single ARM resource, and reduce the complexity associated with common cluster operations such as scaling, and certificate rollovers. In this release we are bringing additional capabilities such as availability zone spanning, stateless node types, application secrets, application managed identities, and the ability to configure cluster upgrades.
+
+To utilize these new features, you must update the API version in the ARM template to *2021-05-01*.
+
+In this release, Service Fabric managed clusters are now available in all Azure public cloud regions and will roll out to Azure special clouds in the coming weeks.
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:7]: Per https://azure.microsoft.com/en-us/global-infrastructure/geographies/ I think we should say this instead.
In this release, Service Fabric managed clusters are now available in all Azure public cloud regions and will roll out to Azure Government regions in the coming weeks.

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* [Stateless node types](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-stateless-node-type) - Stateless node types enable faster node scaling, and VM patching when running services that do not require maintaining any state.
+* [Application secrets](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-configuration#enable-automatic-os-image-upgrades) - You can now access encrypted secrets from your Service Fabric application.
+* [Application managed identities](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-application-managed-identity.md) - Application managed identities enable you to access other Azure resources from your application.
+* [Cluster upgrade configurations](https://docs.microsoft.com//azure/service-fabric/service-fabric-cluster-upgrade) - You can now select between manual, and automatic upgrades and configure an upgrade window to meet the needs of your environment.
+
+## Upcoming Features
+
+* Bring your own VNET.
+* Multiple frontend IPs.
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:30]: I don't think we should commit to this one yet

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+| API |
+|---------|
+| Microsoft.ServiceFabric/managedclusters |
+| Microsoft.ServiceFabric/managedclusters/nodetypes |
+| Microsoft.ServiceFabric/managedclusters/applications |
+| Microsoft.ServiceFabric/managedclusters/applicationTypes |
+| Microsoft.ServiceFabric/managedclusters/applications/services |
+
+## Features
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:19]: ## New Features with this GA release ? 
Just ## features reads like "everything" which I know isn't the intention

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* [Stateless node types](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-stateless-node-type) - Stateless node types enable faster node scaling, and VM patching when running services that do not require maintaining any state.
+* [Application secrets](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-configuration#enable-automatic-os-image-upgrades) - You can now access encrypted secrets from your Service Fabric application.
+* [Application managed identities](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-application-managed-identity.md) - Application managed identities enable you to access other Azure resources from your application.
+* [Cluster upgrade configurations](https://docs.microsoft.com//azure/service-fabric/service-fabric-cluster-upgrade) - You can now select between manual, and automatic upgrades and configure an upgrade window to meet the needs of your environment.
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:25]: typo in url //

You can now select between automatic(default) or manual upgrades and configure an upgrade window to meet the needs of your environment.

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+|---------|
+| Microsoft.ServiceFabric/managedclusters |
+| Microsoft.ServiceFabric/managedclusters/nodetypes |
+| Microsoft.ServiceFabric/managedclusters/applications |
+| Microsoft.ServiceFabric/managedclusters/applicationTypes |
+| Microsoft.ServiceFabric/managedclusters/applications/services |
+
+## Features
+
+* [Availability zone spanning](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-availability-zones) - You can now deploy Service Fabric managed clusters across availability zones to increase resiliency against zone failures.
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:21]: Do we want to align this with the marketing speak?
You can now deploy Service Fabric managed clusters across availability zones to increase resiliency against zone failures.

You can now deploy Service Fabric managed clusters across availability zones to eliminate single points of failure. (going off memory, I think its on our doc page)

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+* [Stateless node types](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-stateless-node-type) - Stateless node types enable faster node scaling, and VM patching when running services that do not require maintaining any state.
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:22]: no comma after scaling

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+* [Application secrets](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-configuration#enable-automatic-os-image-upgrades) - You can now access encrypted secrets from your Service Fabric application.
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:23]: wrong document link ?

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+# Microsoft Azure Service Fabric managed clusters general availability release
+
+Azure Service Fabric managed clusters are now generally available. We first introduced Service Fabric managed clusters in October, 2020. Service Fabric managed clusters enable customers to deploy Service Fabric clusters as a single ARM resource, and reduce the complexity associated with common cluster operations such as scaling, and certificate rollovers. In this release we are bringing additional capabilities such as availability zone spanning, stateless node types, application secrets, application managed identities, and the ability to configure cluster upgrades.
```
Comment (craftyhouse) [inline sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:3]: Not sure I like marketing speak in release notes. I'd prefer more to the point like this? thoughts?

Azure Service Fabric managed clusters are now generally available. In this release we are bringing additional capabilities such as availability zone spanning, stateless node types, application secrets, application managed identities, and the ability to configure cluster upgrades.

idea, pull in/up the region availability statement?

Comment (peterpogorski) [reply sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:3]: I agree this sounds better. I also moved up the region statement.

File: sfmc_release_notes/SF_managed_clusters_ga_release_notes.md (hunk 7 above)
Comment (peterpogorski) [reply sfmc_release_notes/SF_managed_clusters_ga_release_notes.md:23]: Which link?

**Pull request merged**